src/models/subscriptions.js

```diff
@@ -31,9 +31,9 @@ function * setSubscription (subscription, requestingUser) {
 
   let existed
   yield db.transaction(function * (transaction) {
-    const duplicate = yield db.getMatchingSubscription(
+    const duplicates = yield db.getMatchingSubscriptions(
```

Comment:
Since we throw an error when `getMatchingSubscriptions` returns any record (so we never expect it to return more than 1), I'd probably change the name back to `getMatchingSubscription`. This logic could be cleaned up in general, but that's outside the scope of this PR.